### ISSUE TITLE
How is the accurate?

### ISSUE THREAD

**Issue by tranducanhbk** [mks0601/Hand4Whole_RELEASE#4]
Hi mks0601,
I have jsut finish train Hand4Whold follow your instruction with dataset H36M+MPII+MSCOCO 
but when i run demo the results is not good. U can see images below.
I trained 6 epochs eachs state
![input](https://user-images.githubusercontent.com/54969565/165738073-0719b1ce-3b74-4c24-bdd6-02ed589efe21.png)
![render_original_img](https://user-images.githubusercontent.com/54969565/165738081-5e08e3fb-515e-4a86-8e29-c0e750b777f8.jpg)

**Comment by mks0601** [comment]
Did you set bbox in the demo code correctly?

**Comment by tranducanhbk** [comment]
Could u tell me detail about this parameter and how to set it correctly?

**Comment by tranducanhbk** [comment]
after i set bbox corectly the hand is good but leg and feet is wrong
![Uploading render_original_img.jpg…]()

**Comment by tranducanhbk** [comment]
I compared my model and your model and the result is below.
this is mine:
![render_original_img](https://user-images.githubusercontent.com/54969565/165775119-6099263d-3a57-4a68-acb1-f41b215ec9b4.jpg)
and this is yours (i got your model from this link Download Hand4Whole trained on H36M+MPII+MSCOCO from [here](https://drive.google.com/file/d/1fHF_llZSxbjJNL_Gsz_NbiWAfb1fSsNZ/view?usp=sharing).)
![render_original_img](https://user-images.githubusercontent.com/54969565/165776318-3855a325-cbda-443e-8e10-f1b1a25cb21a.jpg)
the result is same

**Comment by mks0601** [comment]
Oh I see. I might be confused with my other models with better performances. I guess the wrong result of the right leg is due to the occlusion.

**Comment by mks0601** [comment]
That will be submitted to another conference. The codes and pre-trained models of that paper will be released upon acceptance. Please stay tuned :)

**Comment by tranducanhbk** [comment]
Hi mks0601!
When i test my model trained with H36M+MPII+MSCOCO dataset but the result i not good so i would like to make custom dataset to train.
Could u give detail guide for make custom data to train
this is result
![input1](https://user-images.githubusercontent.com/54969565/166446324-64ac64b2-34dd-42ff-8412-7983acf917e6.png)
![render_original_img](https://user-images.githubusercontent.com/54969565/166446330-ddf02b97-41e8-4d1a-b3ce-a72e2d88bb72.jpg)

**Comment by mks0601** [comment]
Please prepare your custom dataset in a MSCOCO format (http://cocodataset.org/#format-data).
And make a dataloader like https://github.com/mks0601/Hand4Whole_RELEASE/blob/main/data/MSCOCO/MSCOCO.py

**Comment by tranducanhbk** [comment]
thank you for your quickly reply

**Comment by tranducanhbk** [comment]
Hi mks0601!
I have just make a custom dataset follow COCO format  (just only label for boby part) then I train HAND4WHOLE with H36M+MPII+MY_MSCOCO dataset but when i run demo the error occur like images below
![loi](https://user-images.githubusercontent.com/54969565/167530442-9a32a722-93cf-43f2-9221-19abcfd09aeb.JPG)
Do u have any ideal for this error? I have to label face, hand and foot part or not?

**Comment by mks0601** [comment]
I guess you should check the dimension of the mesh. Without face/hand/foot labels, the output mesh will have weird face/hand/foot

**Comment by tranducanhbk** [comment]
I checked mesh output all element is nan
![loi](https://user-images.githubusercontent.com/54969565/167541931-7d10c9d7-a13c-48f1-8c52-b690f67a52fd.JPG)
Some thing wrong when we train with COOC data withoit face,hand and foot labeling?

**Comment by tranducanhbk** [comment]
this is my custom json file for trainning
[coco_wholebody_train_v1.0.txt](https://github.com/mks0601/Hand4Whole_RELEASE/files/8657393/coco_wholebody_train_v1.0.txt)
if u found any abnormal please tell me

**Comment by tranducanhbk** [comment]
Do u have any update for this issue?
I am looking forwarding your reply!
Thank you

**Comment by mks0601** [comment]
Sorry I have no time to check the details of your custom dataset.
MSCOCO should have body/hand/foot/face annotations. If you want to get the body-only model, check [here](https://github.com/mks0601/Hand4Whole_RELEASE/tree/Pose2Pose)

**Comment by tranducanhbk** [comment]
maybe u misunderstand my issue, I would like to made HAND4WHOLE version but when I test some image in my data with weight  trained public dataset (H3.6,MSCOO,MPII) the result is not good. So I would like to make custom data from my data to train. I think I got mistakes when making custom dataset. In your dataset have MSCOCO_train_SMPLX_all_NeuralAnnot.json but in m data don't have. if u have time, please tell me how to create MSCOCO_train_SMPLX_all_NeuralAnnot.json for custom dataset
Many Thanks

**Comment by mks0601** [comment]
without SMPLX parameters, the model will perform really bad. They are 3D pseudo-GTs of MSCOCO dataset. My another paper (NeuralAnnot) is used to obtain the 3D pseudo-GTs.

**Comment by tranducanhbk** [comment]
thank you so much
